.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,137 @@
+on:
+  # Branch deletion
+  delete:
+  pull_request:
+  push:
+    branches:
+      - "master"
+
+env:
+  PRODUCTION_BRANCH: "master"
+  REPO_NAME: ${{ github.event.repository.name }}
+  PULL_REQUEST_NUMBER: ${{ github.event.number }}
+  GITHUB_ACTION: ${{ github.event.action }}
+  AWS_REGION: "us-east-1"
+  SERVICE_NAME: "badges"
+  REPO_OWNER: "FormidableLabs"
+  GITHUB_HASH: ${{ github.event.after }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Might be good to consider putting this just in the env of the tasks that need it to avoid all tasks having access to the secret.

Reply:
Makes sense, thanks